pkg/certs/cert-inspection/certgraphanalysis/collection_options.go

```diff
@@ -44,3 +52,8 @@ func isRevisioned(ownerReferences []metav1.OwnerReference) bool {
 
 	return false
 }
+
+func hasMonitoringHashLabel(labels map[string]string) bool {
+	_, ok := labels["monitoring.openshift.io/hash"]
```

Comment:
you confirmed this hash isn't present on the original secret and configmap?

Reply:
Yeah, these are added only for the generated configmaps/secrets